appveyor.yml

```diff
@@ -55,7 +55,7 @@ nuget:
 install:
 
   ## Use Chocolatey to install SWIG.
-  - choco install swig --version 3.0.6 --yes --limitoutput
+  - choco install swig --version 3.0.6 --yes | out-null
```

Comment:
I think this requires running this command with powershell (`ps:` at the start).